cpp/ql/test/library-tests/dataflow/taint-tests/taint.expected

```diff
@@ -0,0 +1 @@
+| vector.cpp:392:7:392:8 | v9 | Unexpected result: ir= |
```

Comment:
The inline expectation for this should be corrected.

Reply:
Oops. Missed that one. Fixed in https://github.com/github/codeql/pull/12316/commits/b36d4931f217f92f0090f98c46b03f54f02a0b26. Thanks!